book/src/bridge-module/opaque-types/README.md

```diff
@@ -240,4 +240,45 @@ if val1 == val2 {
 } else {
     print("Not equal")
 }
+```
+
+#### #[swift_bridge(Hashable)]
+
+The `Hashable` attribute allows you to expose a Rust `Hash` trait via Swift's
+`Hashable` protocol.
+
+```rust
+#[swift_bridge::bridge]
+mod ffi {
+    extern "Rust" {
+        #[swift_bridge(Hashable)]
+        type RustHashType;
+
+        #[swift_bridge(init)]
+        fn new(num: u32) -> RustHashType;
+    }
+}
+
+#[derive(Hash, PartialEq)]
+struct RustHashType(u32);
+
+impl RustHashType {
```

Comment:
You can remove this `impl`, as well as the one in the `Equatable` example above.

I know I'm the one that introduced this, but I'm now realizing that it's a waste of space.

You can also delete the `fn new` in the bridge module in both this Hashable example as well as the `Equatable` one. Just to get rid of noise.